tests/conftest.py

```diff
@@ -38,6 +38,12 @@ def metrics_sample():
     )
 
 
+@pytest.fixture()
+def coordinator_metrics_sample():
+    """Return a valid coordinator metric object."""
+    return {"state": 1, "round": 2, "number_of_selected_participants": 0}
```

Comment:
the participant's metrics are transformed into a json at the participant's side to have an easily shippable data format that can be send as a protobuf message to the coordinator. the coordinator's metrics on the other of course don't have to be wrapped, since they can directly be put into the db.

Reply:
Right, and the reason why we are using JSON is that influxdb allows multiple types (int, float, string) for a `fields` value.

e.g.

```
{
	"measurement": "CPU utilization",
	"time": 1234326435,
	"tags": {"id": "127.0.0.1:1345"},
	"fields": {
		"CPU_1": 90.8, # float
		"CPU_2": 90, # int
		"CPU_3": "23", # string
		...
	},
},
```

In protobuf we can't have multiple types for a value. As a workaround we took JSON but maybe we can find a better solution in the future.

Reply:
thanks, both.